etl/src/main/scala/etl/RecipeParsing.scala

```diff
@@ -117,4 +132,19 @@ object RecipeParsing {
 
   }
 
+  private object NumberedParagraphs {
```

Comment:
Could call this `NumberedParagraph` to be consistent with the other matchers. It only matches one paragraph.